src/Writer/AbstractWriter.php

```diff
@@ -103,16 +110,7 @@ final public function addRow(Row $row): void
             throw new WriterNotOpenedException('The writer needs to be opened before adding row.');
         }
 
-        try {
-            $this->addRowToWriter($row);
-        } catch (OpenSpoutException $e) {
-            // if an exception occurs while writing data,
-            // close the writer and remove all files created so far.
-            $this->closeAndAttemptToCleanupAllFiles();
```

Comment:
It turned out that all this code where only involved with invalid Cell value types